src/display/text_layer.js

```diff
@@ -226,7 +529,54 @@ var renderTextLayer = (function renderTextLayerClosure() {
           self._renderTimer = null;
         }, timeout);
       }
-    }
+    },
+    _expandTextDivs: function TextLayer_expandTextDivs(expand) {
+      if (expand) {
+        for (var i = 0, ii = this._textDivs.length - 1; i < ii; i++) {
```

Comment:
It looks like this loop will skip the *last* element in the array, I think it must be like this instead:
```javascript
for (var i = 0, ii = this._textDivs.length; i < ii; i++) {
```